src/herder/test/UpgradesTests.cpp

```diff
@@ -315,7 +315,7 @@ testValidateUpgrades(VirtualClock::time_point preferredUpgradeDatetime,
                      bool canBeValid)
 {
     auto cfg = getTestConfig();
-    cfg.LEDGER_PROTOCOL_VERSION = 10;
+    cfg.LEDGER_PROTOCOL_VERSION = 11;
```

Comment:
I don't think you needed to update these tests (or maybe they should have been written parametrized in the first place without the hardcoded numbers)

Reply:
fixed.